### PR TITLE
Adding pymatreader

### PR DESCRIPTION
**Pull request by larsoner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pymatreader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+about:
+  home: https://gitlab.com/obob/pymatreader
+  summary: Convenient reader for Matlab mat files
+  license: BSD-2-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - thht
```
Comment (larsoner) [inline recipes/pymatreader/meta.yaml:47]: @thht can you:

>  [post] a comment confirming [you] are willing to be listed there.

? Or if you don't want to co-maintain the recipe I can remove you. Usually there is little to nothing to do -- once something is pushed to PyPi, a bot notices, opens a PR to update the recipe, auto-merges, and auto-rebuilds. In my experience you only really have to change things when dependencies change

**Comment by hoechenberger** [comment]
@larsoner If you're looking for co-maintainers, feel free to add me. No hard feelings if you don't, though :)

**Comment by thht** [comment]
thanks a lot @larsoner for adding the recipe!
i confirm that i am willing to be listed as a co maintainer.

**Comment by larsoner** [comment]
@conda-forge/staged-recipes, this one seems to be ready for review. Thanks!

**Comment by jakirkham** [comment]
Thanks all! 😄

**Pull request merged**